server/metrics/metrics.go

```diff
@@ -93,6 +94,9 @@ func SchemaUpdateRepaired(project string, branch string, collection string) {
 }
 
 func InitializeMetrics() func() {
+	if metricsInitialized {
```

Comment:
What about instead of this do,
```
var once sync.Once

InitializeMetrics() {
    once.Do(func() {})
}
```

Reply:
I added a mutex, which is basically the same as what this does internally. We need to pass `io.Closer` outside.

Reply:
You can have io.Closer outside of `Do()`